resdx/dx_unit.py

```diff
@@ -188,17 +188,17 @@ def check_array_order(self, array):
     if not all(earlier >= later for earlier, later in zip(array, array[1:])):
       sys.exit(f'Arrays must be in order of decreasing capacity. Array items are {array}.')
 
-  def make_condition(self, condition_type, compressor_speed=0, indoor=None, outdoor=None):
+  def make_condition(self, condition_type, compressor_speed=0, indoor=None, outdoor=None, air_vol_flow_per_rated_cap=None, air_vol_flow=None):
     if indoor is None:
       indoor = condition_type().indoor
     if outdoor is None:
       outdoor = condition_type().outdoor
 
     condition = condition_type(indoor=indoor, outdoor=outdoor, compressor_speed=compressor_speed)
     if condition_type == CoolingConditions:
-      condition.set_rated_air_flow(self.flow_rated_per_cap_cooling_rated[compressor_speed], self.net_total_cooling_capacity_rated[compressor_speed])
+      condition.set_rated_air_flow(self.flow_rated_per_cap_cooling_rated[compressor_speed], self.net_total_cooling_capacity_rated[compressor_speed],air_vol_flow_per_rated_cap, air_vol_flow)
```

Comment:
You need to make sure only one of air_vol_flow_per_rated_cap or air_vol_flow is set and use it to set the air_vol_flow argument to the function (don't pass them both all the way through).